samples/contrib/aws-samples/README.md

```diff
@@ -0,0 +1,209 @@
+# Sample AWS SageMaker Kubeflow Pipelines 
+
+This folder contains many example pipelines which use [AWS SageMaker Components for KFP](https://github.com/kubeflow/pipelines/tree/master/components/aws/sagemaker). Following sections explain the setup you need to run these pipelines. Once you are done with the setup, [simple_train_pipeline](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/aws-samples/simple_train_pipeline) is a good place to start if you have never used these components before.
+
+
+
+## Prerequisites 
+
+1. You need a cluster with Kubeflow installed on it. [Install Kubeflow on AWS cluster](https://www.kubeflow.org/docs/aws/deploy/install-kubeflow/)
+2. Install the following on your local machine or EC2 instance (These are recommended tools. Not all of these are required)
+    1. [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html). If you are using an IAM user, configure your [Access Key ID, Secret Access Key](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) and preferred AWS Region by running:
+       `aws configure`  
+    2. [aws-iam-authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html) version 0.1.31 and above  
+    3. [eksctl](https://github.com/weaveworks/eksctl) version above 0.15  
+    4. [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl) version 1.17 and above  
+    5. [KFP SDK](https://www.kubeflow.org/docs/pipelines/sdk/install-sdk/#install-the-kubeflow-pipelines-sdk) (installs the dsl-compile and kfp cli)
+    6. [jq](https://stedolan.github.io/jq/download/)
+
+
+## IAM Permissions 
+
+You need two IAM Roles/Users 
+1. For KFP pods to access AWS SageMaker. Let's call this one **kfp-pod-permissions**
+2. For SageMaker job to access S3 buckets and other SageMaker services. Let's call this one **kfp-sagemaker-execution-permissions**
```

Comment:
instead of talking about execution role here talk about two options and lets move execution role later.